lib/submissions.spec.ts

```diff
@@ -12,6 +13,30 @@ const mockedAxios = axios as jest.Mocked<typeof axios>;
 
 const { ENDPOINT_API, AWS_KEY } = process.env;
 
+describe('getUnfinishedSubmissions', () => {
+  it('should work properly', async () => {
```

Comment:
We can do better with the test names here

```suggestion
  it('should return a list of incomplete submissions', async () => {
```